tests/unit/test-safe-svg.php

```diff
@@ -262,6 +262,53 @@ public function test_one_pixel_fix() {
 							),
 						),
 					),
+					array(
```

Comment:
Is this what `wp_get_attachment_metadata()` is expected to return following your changes in this and the other PR? 

Using `svgNoDimensions.svg` I don't get a sizes array stored.

Reply:
@peterwilsoncc I have updated this as per the actual response from the WP for the `svgNoDimensions.svg` file.